### ISSUE TITLE
Introduce IO Helper

### ISSUE THREAD

**Issue by bcremer** [shopwareLabs/sw-cli-tools#1]
tl;dr: Too many IO relates method params.

As of Symfony 2.5 the [Dialog Helper](http://symfony.com/doc/current/components/console/helpers/dialoghelper.html) is deprecated in favour of the new [Question helper](http://symfony.com/doc/current/components/console/helpers/questionhelper.html) and will eventually removed in Syfony 3.0.

I already started the replacement in the following commit: https://github.com/ShopwareAG/sw-cli-tools/commit/dee8fd7808090670c69652d954217080386df07a.
## Problem

The new Question helper does not have a reference to the InputInterface so we have to introduce another `$input` param in many methods.

Old:

```
$interactionManager = new PluginOperationManager(
    $pluginManager, 
    $pluginSelector,
    $dialogHelper, 
    $output, 
    $utilities,
)
```

New

```
$interactionManager = new PluginOperationManager(
    $pluginManager, 
    $pluginSelector,
    $questionHelper, 
    $output, 
    $input,
    $utilities,
)
```

In my opinion it's cumbersome to have so many IO related method params in sometimes deeply nested method calles.
## Proposal

I would like to introduce a IO Helper that abstracts IO related tasks like input/output questions, dialogs in a simple interface. 

We should have a closer look on Composer´s [IOInterface](https://github.com/composer/composer/blob/master/src/Composer/IO/IOInterface.php) for a possible solution to that problem.

Proposed

```
$interactionManager = new PluginOperationManager(
    $pluginManager, 
    $pluginSelector,
    $ioInterface,
    $utilities,
)
```

/ping @dnoegel
